docs/content/preview/manage/backup-restore/lightweight-db-clone.md

```diff
@@ -0,0 +1,258 @@
+    ```sh
+    ./bin/yb-admin --master_addresses ip1:7100,ip2:7100,ip3:7100 create_snapshot_schedule 1440 4320 ysql.production_db
+    ```
+
+1. Create a table `t1` and add some data as follows:
+
+    ```sql
+    \c production_db;
+    production_db=# CREATE TABLE t1 (k INT, v INT);
+    production_db=# INSERT INTO t1 (k,v) SELECT i,i%2 FROM generate_series(1,5) AS i;
+    production_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+    k | v
+   ---+---
+    1 | 1
+    2 | 0
+    3 | 1
+    4 | 0
+    5 | 1
+   (5 rows)
+    ```
+
+1. Determine the exact time when your database is in the correct state. This timestamp will be used to create a clone of the production database from the point when it was in the desired state. Execute the following SQL query to retrieve the current time in UNIX timestamp format:
+
+    ```sql
+    production_db=# SELECT (EXTRACT (EPOCH FROM CURRENT_TIMESTAMP)*1000000)::decimal(38,0);
+    ```
+
+    ```output
+         numeric
+     ------------------
+      1723243720285350
+      (1 row)
+    ```
+
+1. Drop the table `t1` simulating a user error dropping the table accidentally.
+
+    ```sql
+    production_db=# DROP TABLE t1;
+    ```
+
+    ```output
+    DROP TABLE
+    ```
+
+    ```sql
+    production_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+    ERROR:  relation "t1" does not exist
+    LINE 1: SELECT * FROM t1 ORDER BY k;
+    ```
+
+   The table is dropped and there is no way you can query it.
+
+1. Create a database `clone_db` using `production_db` as the template and using the timestamp generated in step 4.
+
+    ```sql
+    production_db=# CREATE DATABASE clone_db TEMPLATE production_DB AS OF 1723243720285350;
+    ```
+
+    ```sql
+    \c clone_db
+    ```
+
+    ```output
+    You are now connected to database "clone_db" as user "yugabyte".
+    ```
+
+    ```sql
+    clone_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+     k | v
+    ---+---
+     1 | 1
+     2 | 0
+     3 | 1
+     4 | 0
+     5 | 1
+    (5 rows)
+    ```
+
+    `clone_db` contains all the data from `production_db` at the specified timestamp, which means you can read table `t1` that was dropped accidentally. You now have two isolated databases that can serve reads and writes independently. You can copy the data back to `production_db` or switch the workload to `clone_db`.
+
+## Best practices
+
+Although creating a clone database is quick and initially doesn't take up much added disk space as no data is copied, it's important to understand that a clone does create an independent set of logical tablets. This effectively doubles the number of tablets, although the clone database tablets share the same data files as the original database. Keep in mind the following impacts:
+
+- Higher CPU usage due to the additional tablets
+- Increased memory consumption from the extra tablets
+- Potentially up to twice the disk usage (after major compactions) due to the retention of older data versions
```

Comment:
```suggestion
- Increased space usage only if compaction is performed on clone or original database. This is because both original and post-compaction data files are kept in disk i.e.: say compaction is performed on the source database which generates new compacted files. The old data files are kept to serve reads for the clone database.
```